_includes/head.html

```diff
@@ -42,4 +42,5 @@
   </script>
   <link rel="stylesheet" href="{{ '/assets/css/style.css' | relative_url }}" />
   {%- feed_meta -%}
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
```

Comment:
Is this `integrity` tag enforced by every major browser? If so, since when? I'd consider the "safe" alternative to include the file in the EIP repo.

Reply:
Integrity is supported by all major browsers except the deprecated IE.

Source: https://caniuse.com/#feat=subresource-integrity